include/dlaf/factorization/qr/t_factor_mc.h

```diff
@@ -155,18 +155,17 @@ hpx::future<matrix::Tile<T, Device::CPU>> trmvUpdateColumn(
     hpx::future<matrix::Tile<T, Device::CPU>>& tile_t) {
   // Update each column (in order) t = T . t
   // remember that T is upper triangular, so it is possible to use TRMV
-  auto trmv_func =
-      hpx::util::unwrapping([](auto&& tile_t, TileElementIndex t_start, TileElementSize t_size) {
-        // clang-format off
+  auto trmv_func = hpx::unwrapping([](auto&& tile_t, TileElementIndex t_start, TileElementSize t_size) {
+    // clang-format off
```

Comment:
note-to-self: check formatting before merging

Reply:
Can we just remove the `// clang-format` comments?

Reply:
For blas/lapack calls which generally have a long list of parameters I find that the manual formatting with `// clang-format` is helpful to increase the readability, but I agree with you all it is a pain to maintain the indentation of such sections of code.

Reply:
@albestro I've tentatively removed the `clang-format` comments (and reformatted accordingly) in the latest commit. Could you please have a look again to see how terrible you think it looks and if it's worth the manual formatting?